src/tools/rados/rados.cc

```diff
@@ -189,6 +189,7 @@ void usage(ostream& out)
 "   -s name\n"
 "   --snap name\n"
 "        select given snap name for (read) IO\n"
+"   -i infile\n"
```

Comment:
could you instead use a long name, like `--input-file`, for better readability if we are going to use this option in scripts. and might want to add a short description for this option?